build.assets/macos/tshdev/README.md

```diff
@@ -36,6 +36,8 @@ $skel/sign.sh tsh                  # sign tsh binary
 security find-identity -vp codesigning      # list codesign identities
 security cms -D -i /path/to/profile         # inspect provisioning profile
 codesign -d --entitlements /path/to/binary  # shows binary entitlements
+codesign --verify --verbose                 # verifies code signatures
+codesign -dv --verbose=4                    # verifies code signatures with verbose output
```

Comment:
I tend to use `codesign -dvv`, I think it might be the same?

Include only the more verbose one here?